linkerd/app/src/lib.rs

```diff
@@ -203,16 +203,13 @@ where
             Conditional::Some(config) => info!("using identity service at {:?}", config.svc.addr),
             Conditional::None(reason) => info!("identity is DISABLED: {}", reason),
         }
-        info!("routing on {:?}", outbound_listener.local_addr());
+        info!("admin on {}", admin_listener.local_addr(),);
         info!(
-            "proxying on {:?} to {:?}",
-            inbound_listener.local_addr(),
-            config.inbound_forward
-        );
-        info!(
-            "serving admin endpoint metrics on {:?}",
-            admin_listener.local_addr(),
+            "tap on {:?}",
+            control_listener.as_ref().map(|l| l.0.local_addr())
         );
+        info!("outbound on {:?}", outbound_listener.local_addr());
+        info!("inbound on {}", inbound_listener.local_addr(),);
```

Comment:
Here as well

Reply:
copypasta. fmt doesn't hit the tracing macros, apparently